helm/templates/tenantservice/tenantservice-deployment.yaml

```
@@ -0,0 +1,98 @@
+  selector:
+    matchLabels:
+      app: tenantservice
+  template:
+    metadata:
+      labels:
+        app: tenantservice
+        tier: backend
+    spec:
+      containers:
+      - name: tenantservice
+        image:  {{ .Values.tenantService.imageName }}:{{ .Values.tenantService.imageVersion }}
```

YAMLlint (1.37.1)

[warning] too many spaces after colon

(colons)